### ISSUE TITLE
Upgrade to okhttp3

### ISSUE THREAD

**Issue by wimdeblauwe** [sendinblue/APIv3-java-library#15]
Is it possible to update the library to use okhttp3 instead of okhttp2 like it currently does. I am using Retrofit (com.squareup.retrofit2:retrofit:jar:2.6.2) already in my application which depends on okhttp3 (com.squareup.okhttp3:okhttp:jar:3.14.4)

sendinblue depends on okhttp2 (com.squareup.okhttp:okhttp:jar:2.7.5) and this gives problems in my application.

Given that okhttp 3 was already [released in 2016](https://square.github.io/okhttp/changelog_3x/#version-300) and there is even a [version 4](https://square.github.io/okhttp/changelog/) already, it might be good to update.

Another alternative might be to shade the dependencies to avoid clashes.

**Comment by amitsendinblue** [comment]
Hi @wimdeblauwe, Thanks for the suggestion, We will try this in next release.

**Comment by cdaller** [comment]
Are there any news about upgrading? At the moment, there is a security issue found in okhttp 2.7.5 which is used in sendinblue: https://nvd.nist.gov/vuln/detail/CVE-2018-14335

**Comment by wujek-srujek** [comment]
Same here, the transitive dependency on 2.7.5 prevents me from using the Sendinblue SDK.

**Comment by SeanAda** [comment]
Yes, the CVE is a big problem and causes many issues inautomatic dependency checks.

**Comment by MaximeCheramy** [comment]
Hi, can you have an update on this please? We use 2 different libraries depending on different versions of okhttp that are not compatible. We will have to stop using the Sendinblue library if this isn't fixed.

**Comment by edwardmp** [comment]
@amitsendinblue bringing this up again. OkHttp 2.75. is six years old.. Can you please consider updating to OkHttp 4 soon?

**Comment by matthiaswronka** [comment]
+1 for an update. There is another CVE regarding Version 2.7.5: https://nvd.nist.gov/vuln/detail/CVE-2021-0341

**Comment by Kool-Hussain** [comment]
Updated in new release (v7.0.0). Please check..!

**Comment by MaximeCheramy** [comment]
This isn't an issue to keep an old version of okio?

```
        <okio-version>1.14.0</okio-version>
```

**Comment by edwardmp** [comment]
@Kool-Hussain please update okio version too
